src/components/button/Button.tsx

```diff
@@ -0,0 +1,11 @@
+import "./Button.scss";
+
+function Button({ socialLink }: { socialLink: string }) {
```

Comment:
a but odd name of the component, I would name it something like SocialLinkButton as you are using <a> element in it

Reply:
I've changed that in #4